src/Navbar.js

```diff
@@ -1,13 +1,31 @@
 import React from 'react';
 import './Navbar.css';
-// import styled from "styled-components";
-
-const Navbar = () => {
-  return (
-    <div className='side-bar'>
-      <h2>Widgets</h2>
-    </div>
-  )
+
+class Navbar extends React.Component {
+  constructor() {
+    super()
+    this.state = { isComponentActive: false}
+    this.onClick = this.onClick.bind(this)
+  }
+
+  onClick () {
```

Comment:
isComponentActive built in react function?

Reply:
No, thats just what I am creating as a value for the state.  It could be called anything, was just used to keep track of if the button was clicked or not.  I am not using this component though because I went in a different direction using the Toggle render prop component